### DISCUSSION TITLE
关系图谱: 文献自动关联

### DISCUSSION THREAD

**Discussion by dualer** [MuiseDestiny/zotero-style#375] [Ideas]
是否可以借用一些工具, 例如connected paper, 还比如[这个网站](https://timwoelfle.github.io/Local-Citation-Network/), 使得文献自动关联? 感觉这样关系图谱才更有价值

**Comment by MuiseDestiny** [comment]
这个功能工作量可以单独做一个插件了。

这些网站就是靠数据盈利的，不是那么轻易获取的。比如cp访客每天五次机会。

没有数据，这个功能谈不上的。

**Reply by dualer** [reply]
那后面那个网站呢?这是免费的 应该还有很多类似的网站

**Reply by MuiseDestiny** [reply]
工作量很大。即使做了也没时间维护的。1是做不到网页那种体验，2是目前cp官方已经联系我做cp的zotero插件作为官方插件。
